### PR TITLE
3 commit

### PR DESCRIPTION
**Pull request by Hueimin**

### CODE REVIEW

**Review by LowSugarCoke — COMMENTED** [review]
Please write lower case

File: Hi.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 hi
 hello
+HEE HA HA
```
Comment (LowSugarCoke) [inline Hi.txt:3]: Please write lower case

Comment (Hueimin) [reply Hi.txt]: finish

Comment (LowSugarCoke) [reply Hi.txt]: Done

**Review by LowSugarCoke — APPROVED** [review]

**Pull request merged**